scylla-cdc/src/reader.rs

```diff
@@ -0,0 +1,266 @@
+use std::cmp::{max, min};
+use std::error::Error;
+use std::time;
+use std::time::SystemTime;
+
+use chrono;
+use futures::stream::StreamExt;
+use scylla::frame::value::Timestamp;
+use scylla::{Session, SessionBuilder};
+use tokio::time::sleep;
+
+type StreamID = Vec<u8>;
```

Comment:
It would be nice if you used the same type as @macher259 
```rust
use crate::stream_generation::StreamID;
```
Note: it requires @macher259 applying this [suggestion](https://github.com/piodul/scylla-cdc-rust/pull/8/files#r747898988) before.